core/src/main/java/org/opensearch/sql/expression/config/ExpressionConfig.java

```diff
@@ -33,8 +37,17 @@ public class ExpressionConfig {
    */
   @Bean
   public BuiltinFunctionRepository functionRepository() {
+
+    QueryContext queryContext = new QueryContext() {
+      private final Clock currentTime = Clock.fixed(Instant.now(),
+          ZoneId.systemDefault());
+      @Override
+      public Clock getQueryStartTime() {
+        return currentTime;
+      }
+    };
```

Comment:
The implication is BuiltinFunctionRepository and QueryContext is created per request. With current Spring configuration, it is true. I think we should add test to guarantee this.

Reply:
Good point -- I will add this test.